pangeo_forge_orchestrator/routers/github_app.py

```diff
@@ -432,17 +485,52 @@ async def handle_pr_comment_event(
             .where(MODELS["recipe_run"].table.recipe_id == recipe_id)
             .where(MODELS["recipe_run"].table.head_sha == pr["head"]["sha"])
         )
-        # TODO: handle error if there is no matching result. this would arise if the slash
-        # command arg was a recipe_id that doesn't exist for this feedstock + head_sha combo.
-        matching_recipe_run = db_session.exec(statement).one()
-        logger.debug(matching_recipe_run)
+
+        try:
+            # Handle the case where the payload was redelivered, and we end up with multile runs
+            # for the same recipe_id and head_sha.
+            matching_recipe_run = db_session.exec(statement).one()
+        except MultipleResultsFound as exc:
+            logger.error(exc)
+            logger.debug(f"Multiple runs found for {recipe_id=} and {pr['head']['sha']=}.")
+            logger.debug("Using the most recent run.")
+            matching_recipe_run = db_session.exec(statement).first()
+
+        except NoResultFound as exc:
+            # handle error if there is no matching result. this would arise if the slash
+            # command arg was a recipe_id that doesn't exist for this feedstock + head_sha combo.
+            output = f"No run found for {recipe_id=} and {pr['head']['sha']=}."
+            logger.debug(output)
+            logger.error(exc)
+
+            matching_recipe_run = None
+
+        if matching_recipe_run is None:
+            # Post a comment and emoji reaction to confirm that the command was received.
+            key = "**The latest updates on recipe(s) in this PR:**"
+            title = """\
+
+| ID | Sync status | Conclusion | URL | Updated |
+|:------:|:------:|:---:|:---:|:-------:|"""
+
+            message = dedent(
+                f"""\
+{key}
+{title}
+| - | ❌ Failed | - | - | {get_last_update_time()} |
+
+{output}
+"""
```

Comment:
Stylistically, what are your feeling about left-justifying multiline strings versus using `dedent` to keep them nested? I had been doing the latter because it seemed more readable to me, but I don't really know what standard practice is here, and don't have super strong feelings either way.

Reply:
i'm going to remove this code block (not used anymore). 

In some cases, the left alignment via dedent didn't work for me
However, I'm happy to follow the same practice you've used throughout the codebase

Reply:
Gotcha.

Curious what cases it did not work in?

Reply:
when i was still embedding markdown tables inside comments, i was getting mixed results (e.g. older versions of this comment: https://github.com/pforgetest/test-staged-recipes/pull/18#issuecomment-1304310464)